packages/eslint-plugin-pf-codemods/src/rules/v6/loginMainFooterLinksItemStructureUpdated/loginMainFooterLinksItem-structure-updated.test.ts

```diff
@@ -0,0 +1,128 @@
+const ruleTester = require("../../ruletester");
+import * as rule from "./loginMainFooterLinksItem-structure-updated";
+
+const loginMainFooterLinksItemError = {
+  message: `LoginMainFooterLinksItem structure has changed. Instead of passing it many properties, everything is now passed directly in a children component.`,
+  type: "JSXElement",
+};
+
+ruleTester.run("loginMainFooterLinksItem-structure-updated", rule, {
+  valid: [
+    {
+      code: `<LoginMainFooterLinksItem />`,
+    },
+  ],
```

Comment:
Could we add some additional valid tests here, for instance when it is a PF imported component but it's being used without the props that have been removed.

Reply:
I think we have to apply a fix in every case it is being imported from PF. There will always be a default `linkComponent` attribute (`a`). So we always have to pass a children wrapped in the component which was internal previously (https://github.com/patternfly/patternfly-react/pull/10107/files)